packages/frontend/src/components/molecules/password-strength-indicator.molecule.tsx

```
@@ -0,0 +1,200 @@
+                    ? 'bg-green-500 dark:bg-green-600'
+                    : 'bg-gray-300 dark:bg-gray-600'
+                )}
+                aria-hidden="true"
+              >
+                {criterion.met && (
+                  <svg
+                    className="h-2.5 w-2.5"
+                    fill="currentColor"
+                    viewBox="0 0 20 20"
+                  >
+                    <path
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)